src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationAsyncResult.cs

```diff
@@ -150,7 +150,12 @@ internal bool CancelStreamOperation()
             if (_callbackInvoked)
                 return false;
 
-            _asyncStreamOperation.Cancel();
+            try
+            {
+                _asyncStreamOperation?.Cancel();
+            }
+            catch { }
```

Comment:
But why is it safe to eat that exception?  In general, we avoid eating all exceptions, an in cases like this typically prefer to just let the process crash.  What exceptions have you seen emerge from this call that you're trying to handle?